File: languages/php/tests/Compat/CompatTest.php
<?php

use Bloock\Bloock;
use Bloock\Client\RecordClient;
use PHPUnit\Framework\TestCase;

final class CompatTest extends TestCase
{
    public static function setUpBeforeClass(): void
    {
        Bloock::$apiKey = getenv("API_KEY");
        Bloock::$disableAnalytics = true;
    }

    public function testCompat()
    {
        $recordClient = new RecordClient();

        $record = $recordClient->fromString("Hello world")->build();
        $hash = $record->getHash();
        $this->assertEquals("ed6c11b0b5b808960df26f5bfc471d04c1995b0ffd2055925ad1be28d6baadfd", $hash);
    }
}

File: languages/java/src/compat/java/CompatTest.java
import static org.junit.jupiter.api.Assertions.assertEquals;

import com.bloock.sdk.client.RecordClient;
import com.bloock.sdk.entity.record.Record;
import org.junit.jupiter.api.Test;

class CompatTest {

  @Test
  void testCompat() throws Exception {
    RecordClient recordClient = new RecordClient();

    Record record = recordClient.fromString("Hello world").build();

    assertEquals(
        record.getHash(), "ed6c11b0b5b808960df26f5bfc471d04c1995b0ffd2055925ad1be28d6baadfd");
  }
}
